### PR TITLE
Add Mapit scripts

### PR DESCRIPTION
**Pull request by alexmuller**
These scripts replace the deployment commands at https://github.gds/gds/puppet/blob/master/modules/mapit/manifests/README.md

### CODE REVIEW

File: mapit.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+@task
+def update_database():
+    """Update a Mapit database using a new database dump"""
+
+    if len(env.hosts) > 1:
+      exit('This command should only be run on one Mapit machine at a time')
+
+    execute(nginx.gracefulstop)
+    execute(app.stop, 'mapit')
+    sudo('service collectd stop')
```
Comment (mattbostock) [inline mapit.py]: Why do we stop collectd?

Comment (jennyd) [inline mapit.py]: It connects to the database and Postgres won't let you drop a database while connections are still open to it.

Comment (mattbostock) [inline mapit.py]: Thanks @jennyd.

**Comment by mattbostock** [comment]
:+1:

**Pull request merged**